pact/message_provider.py

```diff
@@ -115,22 +115,37 @@ def verify(self):
         return_code, _ = verifier.verify_pacts(pact_files, verbose=False)
         assert (return_code == 0), f'Expected returned_code = 0, actual = {return_code}'
 
-    def verify_with_broker(self, enable_pending=False, include_wip_pacts_since=None, **kwargs):
+    def verify_with_broker(
+            self,
+            *pacts,
+            enable_pending=False,
+            include_wip_pacts_since=None,
+            **kwargs
+    ):
         """Use Broker to verify.
 
-        Args:
-            broker_username ([String]): broker username
-            broker_password ([String]): broker password
-            broker_url ([String]): url of broker
-            enable_pending ([Boolean])
-            include_wip_pacts_since ([String])
-            publish_version ([String])
-
+        :param pacts: List of pact to verify. Every pact in that list can be
+            an HTTP URI or a local file.
+        :type pacts: str
+        :param enable_pending: Allow pacts which are in pending state to be
+            verified without causing the overall task to fail.
+        :type enable_pending: bool
+        :param include_wip_pacts_since: Allow pacts that don't match given
+            consumer selectors (or tags) to  be verified, without causing the
+            overall task to fail
+        :type include_wip_pacts_since: str or None
```

Comment:
I don't think we have any formal chosen docstring style, we should discuss this :)

I tend to end up using Google or reST, generally prefer reST, but I think type hints in the code rather than :type gives easier to read/maintain...but let's chat!

Reply:
What is the current style in the project? 

Do either of you have a preference?

Both maintenance for maintainers and end user experience in IDE's is important.

Reply:
The default documentation style of reST in sphinx is supported by IDEs, used by the community and as far as I understand @mikegeeves is OK to forward with it too.